packages/shared/src/components/buttons/ClickableText.tsx

```diff
@@ -21,6 +23,8 @@ function ClickableTextComponent<Tag extends AvailableTags>(
     children,
     tag: Tag = 'button',
     defaultTypo = true,
+    flexRowApplied = true,
+    underlined = true,
```

Comment:
I think we are actually better of keeping this outside the component.
This specific spec guideline uses it, but the overal one doesn't.

You can set classNames for this specific use-case like so:

```
<ClickableText className="flex flex-row">
```

I think you might have noticed it's not inline because the default tag is a button.
You can change it to a link like so:

```
<ClickableText tag="a" />
```

Reply:
Oh and as extra: The underline already is applied to the default ClickableText element on hover.